app-misc/remind/remind-04.01.00.ebuild

```
@@ -1,26 +1,27 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 DESCRIPTION="Ridiculously functional reminder program"
 HOMEPAGE="https://dianne.skoll.ca/projects/remind/"
 SRC_URI="https://dianne.skoll.ca/projects/remind/download/${P}.tar.gz"
```

It looks like everything but the latest directory gets put in this OLD directory :(
```suggestion
SRC_URI="https://dianne.skoll.ca/projects/remind/download/OLD/${P}.tar.gz"
```